### PR TITLE
Ensure `static` variables are considered as declared.

### PR DESCRIPTION
**Pull request by Aerendir**
Issue #1505.

### CODE REVIEW

File: packages/Php/tests/Rector/FunctionLike/AddDefaultValueForUndefinedVariableRector/Fixture/take_static_into_account.php.inc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        static $variable = null;
+        if (null === $variable) {
+            $variable = 'not null anymore';
+        }
+
+        return $variable;
+    }
+}
+
+?>
```
Comment (TomasVotruba) [inline packages/Php/tests/Rector/FunctionLike/AddDefaultValueForUndefinedVariableRector/Fixture/take_static_into_account.php.inc]: If the code should remain untoched, you can remove the part from there onwards (included)

Comment (Aerendir) [reply packages/Php/tests/Rector/FunctionLike/AddDefaultValueForUndefinedVariableRector/Fixture/take_static_into_account.php.inc]: Do you intend all the code after `-----`?

Comment (TomasVotruba) [reply packages/Php/tests/Rector/FunctionLike/AddDefaultValueForUndefinedVariableRector/Fixture/take_static_into_account.php.inc]: Nono, from this line included

Comment (TomasVotruba) [reply packages/Php/tests/Rector/FunctionLike/AddDefaultValueForUndefinedVariableRector/Fixture/take_static_into_account.php.inc]: I've merge it, but see my commit

**Pull request merged**